user-creation/src/main/java/no/unit/nva/useraccessservice/usercreation/person/cristin/CristinPersonRegistry.java

```diff
@@ -221,6 +222,16 @@ private Person asPerson(CristinPerson cristinPerson, CristinCredentials cristinC
                   .map(a -> new Affiliation(a.getKey(), a.getValue()))
                   .collect(toList());
 
+        if (personAffiliations.isEmpty()) {
+            LOGGER.warn("Cristin person {} has no active affiliations", cristinPerson.getId());
+        }
+
+        if (isBlank(cristinPerson.getId())
+            || isBlank(cristinPerson.getFirstname())
+            || isBlank(cristinPerson.getSurname())) {
+            throw new PersonRegistryException("Cristin person is missing required fields");
+        }
+
```

Comment:
nitpick in case you touch this again:

- Maybe useful to include the name of the missing field in the error message?
- Maybe easier to extract some of this validation logic to one or more separate methods/classes?